disqus/public/class-disqus-public.php

```diff
@@ -300,6 +300,11 @@ private function dsq_embed_can_load_for_post( $post ) {
 		if ( 'open' != $post->comment_status ) {
 			return false;
 		}
+		
+		// Don't load embed when comments are closed on a post. These lines can solve a conflict with plugin Public Post Preview.
+		if ( ! comments_open() ){
```

Comment:
Make sure you add the space between the `)` and `{` to fix the PHPCS errors that are failing the build